try{
    throw Error("Muy error")
}
catch(error){
    console.log(error.message)
}
